packages/components/Callout/src/Callout.tsx

```diff
@@ -11,10 +17,14 @@ import { mergeSettings } from '@uifabricshared/foundation-settings';
 import { settings } from './Callout.settings';
 import type { ICalloutProps, ICalloutSlotProps, ICalloutType } from './Callout.types';
 import { calloutName } from './Callout.types';
+import type { NativeProps as CalloutNativeProps } from './CalloutNativeComponent';
+import CalloutNativeComponent from './CalloutNativeComponent';
+import type { NativeProps as MacOSCalloutNativeProps } from './MacOSCalloutNativeComponent';
+import MacOSCalloutNativeComponent from './MacOSCalloutNativeComponent';
 
-const NativeCalloutView = Platform.select({
-  macos: ensureNativeComponent('FRNCallout'),
-  default: ensureNativeComponent('RCTCallout'), // win32
+const NativeCalloutView = Platform.select<HostComponent<MacOSCalloutNativeProps> | HostComponent<CalloutNativeProps>>({
+  macos: MacOSCalloutNativeComponent,
+  default: CalloutNativeComponent, // win32
```

Comment:
or can we not remove component-cache easily because that means double changes in areas like this?